src/Controls/src/Core/Routing.cs

```diff
@@ -142,23 +142,6 @@ public static Element GetOrCreateContent(string route, IServiceProvider services
 			if (s_routes.TryGetValue(route, out var content))
 				result = content.GetOrCreate(services);
 
-			if (result == null)
-			{
-				// okay maybe its a type, we'll try that just to be nice to the user
-				var type = Type.GetType(route);
```

Comment:
I don't think anyone would use a route like:
```
app://home/MyAssemblyName, MyNamespace.MyPageName
```
The space, comma, and dot would make this a very weird URL.